src/components/Card.js

```diff
@@ -15,26 +25,49 @@ class Card {
     return cardElement;
   }
 
-  _handleLike = () => {
+  isLiked() {
+    return this._isLiked;
+  }
+
+  handleLike = () => {
     this._likeButton.classList.toggle("gallery__card-like_active");
   };
 
-  _handleDelete = () => {
+  handleDelete = () => {
     this._element.remove();
```

Comment:
**Could be improved**

The best thing to do after deleting a card is to **remove the link** to the DOM element: `this._element = null;`. It helps javascript garbage collector.

More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management